DevGuide Cleanup and Improvments (#4281)
OpenDDS/OpenDDS: docs/devguide/introduction.rst

#. OpenDDS's native multicast transport does not work with RTPS Discovery due to the way GUIDs are assigned (a warning will be issued if this is attempted).

> For more details in how RTPS discovery occurs, a very good reference to read can be found in :omgspec:`rtps:8.5 Discovery Module`.

Reviewer: ```suggestion
For more details on RTPS discovery, see :omgspec:`rtps:8.5 Discovery Module`.
```
